src/main/java/marquez/api/models/JobRunResponse.java

```diff
@@ -14,23 +14,34 @@
 
 package marquez.api.models;
 
+import java.util.Optional;
 import java.util.UUID;
+import javax.annotation.Nullable;
 import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-import org.hibernate.validator.constraints.NotBlank;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
 
-@Data
 @AllArgsConstructor
-@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
 public final class JobRunResponse {
-  @NotBlank private UUID runId;
+  @Getter @NonNull private final UUID runId;
+  @Nullable private final String nominalStartTime;
+  @Nullable private final String nominalEndTime;
+  @Nullable private final String runArgs;
+  @Getter @NonNull private final String runState;
```

Comment:
^^ seems like we're going beyond just lombok annotations here. But since you're adding JobRun fields in this PR, what about `createdAt`?

Reply:
Sure. But I only annotated the fields and no new fields were added. The current field list is: `runId`, `nominalStartTime`, `runArgs`, and `runState`. I'll added `createdAt` and update the openapi spec